### ISSUE TITLE
Cannot access mapped drive from Cmder

### ISSUE THREAD

**Issue by fnzr** [cmderdev/cmder#1939]
### Purpose of the issue
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question

### Description of the issue 

I have a mapped network drive, `Y:`. As you can see from the screenshot, it is accessible from Command Prompt, but not from Cmder.
Using latest Windows 10, tried installing Cmder both form chocolatey and from releases page.
I made no changes on settings, just ran Cmder after unzipping. 
#649 seems to have had the same problem, but that wasn't very helpful.

![screenshot](https://user-images.githubusercontent.com/5471818/47945463-ee7cd880-dee0-11e8-9f4b-ace0bfdf5844.png)

**Comment by fnzr** [comment]
Well, yeah, I assumed the problem is on my machine because that would be a pretty severe bug. Still, what could possibly be causing it?
By the way, I also tried with Git Bash and it can access `Y` drive normally.

**Comment by daxgames** [comment]
@fnzr Did you possibly map the drive as another windows user id or even as an admin user?  I have seen this behavior before but it should also happen with `cmd.exe` outside Cmder.

**Comment by fnzr** [comment]
hmm by disconnecting Y and then connecting again using `net use` command inside Cmder, I was able to access Y as admin in both cmd and cmder.
I then reconnected Y via Windows Explorer -> Map Network Drive and it seems to be all working now.
It might have something to do with admin user, because as you said, after `net use` I had to use `cmd.exe` as admin for it to work.
Well, I`m not sure what was the problem but seems to be fixed, let's see if it persists after a windows restart.
Thanks for your time.
